terraform/gold-e4ca1d-team/.terraform.lock.hcl

```diff
@@ -0,0 +1,24 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
We might not want these included, might have been part of the local env that came from my local env, not sure